### PR TITLE
#557: Showing alerts instead of action sheets on iPad

### PR DESCRIPTION
**Pull request by bonkey**
#557: Showing alerts instead of action sheets on iPad

### CODE REVIEW

**Comment by bonkey** [comment]
👍

File: AtlasSDK/AtlasSDK/Extensions/UIDevice.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright © 2017 Zalando SE. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIDevice {
```
Comment (bonkey) [inline AtlasSDK/AtlasSDK/Extensions/UIDevice.swift]: Yeah, could be there as well, as it goes to Extensions. But it's anyway `UI`

**Comment by hani-ibrahim** [comment]
👍

File: AtlasSDK/AtlasSDK/Extensions/UIDevice.swift (hunk 1 above)
Comment (bonkey) [inline AtlasSDK/AtlasSDK/Extensions/UIDevice.swift]: Ah, you ask why in AtlasSDK? Well, because it's `Extensions` – default folder is in SDK

Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/Extensions/UIDevice.swift]: but we have `Utils` folder in AtlasUI for that

Comment (bonkey) [inline AtlasSDK/AtlasSDK/Extensions/UIDevice.swift]: I created shared Extensions... Well, maybe Utils is better location actually, as _it's anyway `UI`_

**Comment by bonkey** [comment]
👍

**Comment by bonkey** [comment]
This could be problem with zappr statuses:

> zappr — connect ETIMEDOUT 192.30.253.117:443

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**